tools/logs_agent_QA/testBuilder.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
```

Comment:
Some docstrings here would be super-useful.  This is small enough now that I can guess how it's used, but with the addition of a few more knobs this will be quite difficult to decipher.